pkg/check/smoke/load.go

```diff
@@ -233,15 +233,15 @@ func (c *LoadCheck) Run(ctx context.Context, cluster orchestration.Cluster, opts
 
 func (c *LoadCheck) checkStorageRadius(ctx context.Context, client *bee.Client, maxRadius uint8, wait time.Duration) bool {
 	for {
-		rs, err := client.ReserveState(ctx)
+		statusResp, err := client.Status(ctx)
 		if err != nil {
 			c.logger.Infof("error getting state: %v", err)
 			return false
 		}
-		if rs.StorageRadius < maxRadius {
+		if statusResp.CommittedDepth < maxRadius {
 			return true
 		}
-		c.logger.Infof("waiting %v for StorageRadius to decrease. Current: %d, Max: %d", wait, rs.StorageRadius, maxRadius)
+		c.logger.Infof("waiting %v for StorageRadius to decrease. Current: %d, Max: %d", wait, statusResp.CommittedDepth, maxRadius)
```

Comment:
would be nice to change the logs, wording..etc

Reply:
Is CommittedDepth appropriate?
```go
c.logger.Infof("waiting %v for CommittedDepth to decrease. Current: %d, Max: %d", wait, statusResp.CommittedDepth, maxRadius)
```